app/views/layouts/modals/_kids_context.html.erb

```diff
@@ -10,7 +10,7 @@
         <div class="container-fluid">
           <div class="row">
             <div class="col-md-12 text-box">
-              <img src="/character/kids/kibi_context.svg" class="capital-animation"/>
+              <img id="mu-kids-character-context" class="capital-animation"/>
```

Comment:
Why are we using ids instead of classes here?

Reply:
¯\(°_o)/¯

Reply:
Now that I'm passing the jquery object I really should change this to classes, since jquery will only return the first match when searching by id

Reply:
:+1: